apache/superset
# fix: Navbar styles and Welcome page text

### SUMMARY
It fixes an issue with the text in the Welcome page not appearing as reported here https://github.com/apache/superset/pull/19579. It also enhances the styles to follow our guidelines.

### BEFORE

![image](https://user-images.githubusercontent.com/7409244/162082115-acb4eb1f-bc73-4a4c-b649-e6644910b3df.png)

### AFTER

https://user-images.githubusercontent.com/60598000/162194962-b3bd35b7-6e64-42ad-8e91-3ccc68fbaa68.mp4

### TESTING INSTRUCTIONS
1. Go to the login page and make sure the logo appears in the right size immediately
2. Go to the Welcome page and make sure the text "Home" at the top is displaying correctly.

### ADDITIONAL INFORMATION
